src/mjolnir/landmarks.cc

```diff
@@ -296,6 +318,226 @@ bool BuildLandmarkFromPBF(const boost::property_tree::ptree& pt,
                           callback);
   }
 
+  LOG_INFO("Successfully built landmark database from PBF");
+  return true;
+}
+
+// Find landmarks in the tiles and the edges correlated to each landmark,
+// and return the sequence file name where we wrote the correlations
+void FindLandmarkEdges(const boost::property_tree::ptree& pt,
+                       const std::vector<GraphId>& tileset,
+                       const size_t& thread_number,
+                       const size_t& total_threads,
+                       std::promise<std::string>& seq_file_name) {
+  // Open the database and create a graph reader
+  const std::string dbname = pt.get_child("mjolnir").get<std::string>("landmarks", "");
+
+  LandmarkDatabase db(dbname, true);
+  GraphReader reader(pt.get_child("mjolnir"));
+  // create the sequence file
+  std::string file_name = "landmark_dump_" + std::to_string(thread_number);
+  midgard::sequence<std::pair<GraphId, uint64_t>> seq_file(file_name, true);
+
+  for (size_t i = 0; i < tileset.size(); ++i) {
+    // assign tiles for threads
```

Comment:
i find this comment a bit lacking for someone who doesnt already know what is going on
```suggestion
    // every i'th thread works on every i'th tile
```